javascript/modules/client/README.md

```diff
@@ -441,25 +677,326 @@ console.log(proposals);
+  creatorVote: VoteOption.YES,
+};
 
-### Voting on a multisig proposal
+const estimatedGas = await client.estimation.createProposal(
+  proposalCreationParams,
+);
+console.log(estimatedGas.average); // bigint
+console.log(estimatedGas.max); // bigint
+
+for await (
+  const step of client.methods.createProposal(
+    proposalCreationParams,
+  )
+) {
+  try {
+    switch (step.idx) {
+      case ProposalCreationSteps.CREATING:
+        console.log(step.txHash); // 0xb1c14a49...
+        break;
+      case ProposalCreationSteps.DONE:
+        console.log(step.proposalId); // 0x1234...
+        break;
+    }
+  }
+  catch (err){
+    // ...err
+  }
+}
+```
+
+### Voting on a Multisig proposal
+```ts
+import {
+  ClientMultisig,
+  DaoAction,
+  ICreateProposal,
+  VoteOption,
+  ContextPlugin,
+  VoteProposalStep
+} from "@aragon/sdk-client";
+// context
+const contextPlugin = ContextPlugin.fromContext(context, "0x...")
+const client = new ClientMultisig(contextPlugin);
+
+// gas estimation
+const estimatedGas = await client.estimation.voteProposal(
+  proposalId, VoteOptions.YES
+);
+console.log(estimatedGas.average); // bigint
+console.log(estimatedGas.max); // bigint
+
+for await (const step of client.methods.voteProposal(proposalId, VoteOptions.YES)) {
+  try {
+    switch (step.key) {
+    case VoteProposalStep.VOTING:
+        console.log(step.txHash); // 0xb1c14a49...
+      break;
+    case VoteProposalStep.DONE:
+        console.log(step.voteId); // 0x12345...
+      break;
+    }
+  } catch (err) {
+    // ...
+  }
+}
+```
 
-- **TODO**
+### Loading the list of members (Multisig)
+```ts
+import { ClientMultisig, ContextPlugin } from "@aragon/sdk-client";
+// create client
+const contextPlugin = ContextPlugin.fromContext(context, "0x...")
+const client = new ClientMultisig(contextPlugin);
+
+const members = await client.methods.getMembers();
+
+console.log(members); // ["0x3585...", "0x1235...", "0x6785...",]
 
-### Loading the list of members (multisig)
+```
 
-- **TODO**
+### Loading the a proposal by proposalId (Multisig)
+```ts
+import { ClientMultisig, ContextPlugin } from "@aragon/sdk-client";
+// create client
+const contextPlugin = ContextPlugin.fromContext(context, "0x...")
+const client = new ClientMultisig(contextPlugin);
+
+const proposalId = "0x1234567..._0x0"
+
+const proposal = await client.methods.getProposal(proposalId);
+console.log(proposal);
+  /*
+    {
+      id: "0x1234567890123456789012345678901234567890_0x0",
+      daoAddress: "0x56fb7bd9491ff76f2eda54724c84c8b87a5a5fd7",
+      daoName: "DAO 1",
+      creatorAddress: "0x8367dc645e31321CeF3EeD91a10a5b7077e21f70",
+
+      startDate: <Date>,
+      endDate: <Date>,
+      creatonDate: <Date>,
+
+      title: "New Founding for Lorex Lab SubDao",
+      summary: "As most community members know, Aragon has strived.",
+      proposal: "This is the super important proposal body",
+      resources: [{ url: "https://example.com", description: "Example" }],
+
```

Comment:
We need to differentiate the details of a single proposal vs the data of a proposal list entry